Makefile

```diff
@@ -173,6 +173,26 @@ swagger: config/swagger.yaml $(SWAGGER_CODEGEN_CLI)
 swagger-docs:
 	(cd ./apps/aehttp && $(MAKE) swagger-docs);
 
+SWAGGER_STATUS = $(shell cd /tmp/epoch-api-docs/ && git status --porcelain)
+
+swagger-publish-docs-init:
+	git clone git@github.com:aeternity/epoch-api-docs.git /tmp/epoch-api-docs
+	cp apps/aehttp/priv/swagger-docs/swagger.json /tmp/epoch-api-docs/
+
+swagger-publish-docs: swagger-publish-docs-init
+	cd /tmp/epoch-api-docs/; \
+	git add swagger.json; \
+	git config user.email "circleci@aeternity.com"; \
+	git config user.name "CircleCi Aeternity"; \
+	if [ -z "$(SWAGGER_STATUS)" ]; then \
+		echo "Nothing to commit"; \
+	else \
+		git commit -a -m 'Update epoch docs CircleCI'; \
+		git push origin master; \
```

Comment:
Pushing master to master is OK, but we laso have to tag this repo as well when the main repo is tagged

Reply:
duty noted